front/components/assistant/conversation/AgentAction.tsx

```diff
@@ -0,0 +1,11 @@
+import { isRetrievalActionType } from "@app/types/assistant/actions/retrieval";
```

Comment:
External AgentAction to avoid using RetrievalAction directly in AgentMessage rendering as per specs